rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
+      es_data_dirs: "/var/lib/elasticsearch"
+      es_log_dir: "/var/log/elasticsearch"
+      es_config:
+        node.name: "{{ container_name }}"
+        cluster.name: "openstack"
+        http.port: 9200
+        transport.tcp.port: 9300
+        node.data: true
+        node.master: true
+        bootstrap.memory_lock: true
+        network.host: "{{ container_address }}"
   vars:
     is_metal: "{{ properties.is_metal | default(False) }}"
+    es_scripts: false
+    es_templates: false
+    es_version_lock: false
+    es_heap_size: 1g
+    es_api_host: "{{ container_address }}"
+    es_api_port: 9200
+    es_version: "5.5.0"
```

Comment:
How does this relate to ``elasticsearch_version``? The versions don't match. Is that var used anywhere and does it intentionally not match. Can we please comment things properly to ensure that we understand why things are different.